src/core/style/expressions/expression.js

```diff
@@ -61,6 +61,14 @@ export default class Expression {
         return this._getChildren().map(child => child._getDrawMetadataRequirements()).reduce(schema.union, schema.IDENTITY);
     }
 
+    /**
+     * Pre-rendering routine. Should establish the current timestamp in seconds since an arbitrary point in time as needed.
+     * @param {Number} timestamp
+     */
+    _setTimestamp(timestamp) {
```

Comment:
🌈 if this method is public used from the renderer) why the underscore prefix?

Reply:
Consistency with all the other expression methods that are public to the renderer, yet private to the user.